site2/website/static/js/pjax-api.min.js

```diff
@@ -1,4 +1,4 @@
rse()).dropWhile(e=>!r.compare(e,t.orig)&&!r.compare(e,t.dest)).take(1).filter(e=>!!r.compare(e,t.orig)&&r.compare(e,t.dest)).map(e=>n[e]).map(t=>t?o.Just(new i.Config(c.extend({},e,t))):o.Nothing).extract().reduce((e,t)=>t,o.Nothing)}},{"../../../../lib/router":137,"../../../domain/data/config":99,"spica/assign":5,"spica/maybe":19,"spica/sequence":79}],101:[function(e,t,n){"use strict";Object.defineProperty(n,"__esModule",{value:!0});const r=e("../../../lib/error");n.DomainError=class extends r.PjaxError{constructor(e){super(`Domain: ${e}`)}}},{"../../../lib/error":135}],102:[function(e,t,n){"use strict";Object.defineProperty(n,"__esModule",{value:!0});const r=e("../../../lib/url"),i=e("../../data/model/domain/url"),s=e("../../../lib/dom"),o=e("typed-dom");var c,u;n.RouterEvent=class{constructor(e){this.original=e,this.type=this.original.type.toLowerCase(),this.source=o.currentTargets.get(this.original),this.request=new a(this.source),this.location=new l(this.request.url),Object.freeze(this)}},function(e){e.Anchor=HTMLAnchorElement,e.Form=HTMLFormElement,e.Window=window.Window}(c=n.RouterEventSource||(n.RouterEventSource={})),function(e){e.click="click",e.submit="submit",e.popstate="popstate"}(n.RouterEventType||(n.RouterEventType={})),function(e){e.GET="GET",e.POST="POST"}(u=n.RouterEventMethod||(n.RouterEventMethod={}));class a{constructor(e){this.source=e,this.method=(()=>{if(this.source instanceof c.Anchor)return u.GET;if(this.source instanceof c.Form)return this.source.method.toUpperCase()===u.POST?u.POST:u.GET;if(this.source instanceof c.Window)return u.GET;throw new TypeError})(),this.url=(()=>{if(this.source instanceof c.Anchor)return i.standardizeUrl(this.source.href);if(this.source instanceof c.Form)return this.source.method.toUpperCase()===u.GET?i.standardizeUrl(this.source.action.split(/[?#]/)[0]+`?${s.serialize(this.source)}`):i.standardizeUrl(this.source.action.split(/[?#]/)[0]);if(this.source instanceof c.Window)return i.standardizeUrl(window.location.href);throw new TypeError})(),this.body=(()=>this.source instanceof c.Form&&this.method===u.POST?new FormData(this.source):null)(),Object.freeze(this)}}n.RouterEventRequest=a;class l{constructor(e){this.target=e,this.orig=new r.URL(i.standardizeUrl(window.location.href)),this.dest=new r.URL(this.target),Object.freeze(this)}}n.RouterEventLocation=l},{"../../../lib/dom":134,"../../../lib/url":138,"../../data/model/domain/url":97,"typed-dom":87}],103:[function(e,t,n){"use strict";var r=this&&this.__awaiter||function(e,t,n,r){return new(n||(n=Promise))(function(i,s){function o(e){try{u(r.next(e))}catch(e){s(e)}}function c(e){try{u(r.throw(e))}catch(e){s(e)}}function u(e){e.done?i(e.value):new n(function(t){t(e.value)}).then(o,c)}u((r=r.apply(e,t||[])).next())})};Object.defineProperty(n,"__esModule",{value:!0});const i=e("spica/either"),s=e("./module/fetch"),o=e("./module/update"),c=e("./module/update/content"),u=e("../store/path"),a=e("../data/error");var l=e("./model/eav/entity");n.Rout
```

Comment:
what was changed in this file?

Reply:
Thanks @sijie .
Faild == > Failed
like:
new u.DomainError("Faild to validate the content...